feature/experimental/policy/policy_vrf_selection/otg_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go

```diff
@@ -78,7 +78,7 @@ var (
 
 	atePort2 = attrs.Attributes{
 		Name:    "atePort2",
-		MAC:     "01:00:02:01:01:01",
+		MAC:     "00:12:01:00:00:01",
```

Comment:
I'm curious why you're changing the MAC addresses here? I'm not sure what effect it has on the test

Reply:
sorry for late reply, I was on PTO

cptx(juniper) is not resolving ARP with mac address `01:00:02:01:01:01` on ixia-c , I'm checking internally and with juniper team on this and get back to you. 

![image](https://user-images.githubusercontent.com/48308607/223415220-14dfb2d9-107e-41da-9885-c71fdb914e9d.png)